src/db/nedb.js

```diff
@@ -101,6 +129,17 @@ class DBHelper {
       throw err;
     }
   }
+
+  static async isPreviousCreateEventPending(db) {
+    try {
+      const approve = await db.count({ type: 'APPROVECREATEEVENT', status: 'PENDING' });
```

Comment:
`{ type: { $in: ['APPROVECREATEEVENT', 'CREATEVENT'}, status: 'PENDING', 'senderAddress': creator_address}`

Reply:
good call on the senderAddress 👍